source/sync/storageformat5.rst

```diff
@@ -41,25 +41,28 @@ bundle is combined with a per-record 16 byte IV and a user's data is converted
 into ciphertext. The ciphertext is *signed* with the key bundle's **HMAC key**.
 The *ciphertext*, *IV*, and *HMAC value* are then uploaded to the server.
 
-When Sync is initially configured, that client generates a random 128 bit
-sequence called the **Sync Key**. This private key is used to derive a special
-*key bundle* via HKDF. This is called the **Sync key bundle**. The *Sync key
-bundle* is used to encrypt and decrypt a special record on the server which
-holds more *key bundles*. *Key bundles* inside this record are what's used
-to encrypt and decrypt all other records on the server.
+When Sync is initially configured by signing in with a Firefox Account, the
+client obtains a 256-bit encryption key called **kB**. This key is used to
+derive a special *key bundle* via HKDF. This is called the **Sync Key
+Bundle**. The *Sync Key Bundle* is used to encrypt and decrypt a special record
+on the server which holds more *key bundles*. *Key bundles* inside this record
+are what's used to encrypt and decrypt all other records on the server.
+
+
 
 Terminology
 -----------
 
-Sync Key
-    128 bit random value which effectively serves as the master key to Sync.
+kB
```

Comment:
This is an odd choice because it is almost exactly like `KB` for `KiloBytes`. Though wikipedia actually shows `kB` for kilobyte in decimal form as well (while KB is for a kilobyte in binary). Can we not use an overloaded term for this?

Reply:
Hrm. This is a good point, especially when you see it in isolation like this. However, `kB` is the name of record for this thing throughout the FxA documentation, so changing it here risks muddying the waters for folks trying to understand the whole system.

I guess we could call it "FxA master key" or something here, and add a note that FxA docs refer to it as "kB". What do you think @bbangert?

Reply:
I believe that this value will appear throughout FxA docs as well as its an important value. It would be good if this term was incorporated into the Fxa ecosystem glossary. Perhaps @janiceahn would have some good input on this as she helped defined the glossary terminology.

Reply:
Sorry, I clearly didn't fully grok your reply before replying. If it's referred to as `kB` throughout the FxA docs already, that does indeed seem confusing. Is calling it a `FxAKb` too much? Or something just semi-unique that doesn't look exactly like an existing well used tech acronym?

Reply:
I'm going to try out "Class-B Master Key" which seems inline with the existing terminology used in https://github.com/mozilla/fxa-auth-server/wiki/onepw-protocol

Reply:
Also, that document is awesome. It would be great to see that updated (it mentions something launching in 2014).

Reply:
yep; it's one of the thing I'd like to migrate out of a wiki and into docs in github